warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -657,3 +664,49 @@ models:
           - relationships:
              to: ref('dim_services')
              field: key
+  - name: bridge_contracts_x_components
+    description: |
+      Mapping table between contracts and components.
+      Relationship indicates that contract covers the given component.
+      This is a versioned, many-to-many relationship.
+      Joins via this table can result in fanout.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - contract_key
+            - component_key
+    columns:
+      - name: contract_key
+        tests:
+          - relationships:
+             to: ref('dim_contracts')
+             field: key
+      - name: component_key
+        tests:
+          - relationships:
+             to: ref('dim_components')
+             field: key
+  - name: bridge_schedule_dataset_for_validation
+    description: |
+      Mapping table between GTFS RT datasets and the schedule dataset that is used to validate it
+      (i.e., the schedule dataset that the RT dataset references.)
+      This is a versioned relationship.
+      While an RT dataset can only use one schedule at a time, because the records are versioned independently,
+      a given `gtfs_dataset_key` can appear multiple times in this table with different versions of its accompanying schedule.
+      Thus, joins via this table can result in fanout if they don't limit to validity on a specific date.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - gtfs_dataset_key
+            - schedule_to_use_for_rt_validation_gtfs_dataset_key
+    columns:
+      - name: gtfs_dataset_key
+        tests:
+          - relationships:
+             to: ref('dim_gtfs_datasets')
+             field: key
+      - name: schedule_to_use_for_rt_validation_gtfs_dataset_key
+        tests:
+          - relationships:
+             to: ref('dim_gtfs_datasets')
+             field: key
```

Comment:
nit: would it make sense to use https://github.com/dbt-labs/dbt-utils#relationships_where-source to test gtfs_dataset_key only ever points to RT and schedule_to_use_for_rt_validation_gtfs_dataset_key only ever points to Schedule?